modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with condition type, informing you of potential impacts to upgrading your cluster. 
```

Comment:
I think reading this without already knowing the context, I don't immediately get what condition type is supposed to be from this first sentence. Is it a property, a value, or is it more like "the condition type of the operator", without any specific noun/syntax used to describe it? Can it be more specific?

I'd suggest changing `revealed with condition type` to `revealed through their condition type` if condition type isn't described by another noun.

Reply:
Also, sort of a nitpick, but maybe there is a stronger word choice than "revealed", such as "shown" or "displayed". Not sure though, feel free to ignore this point.